docs/pref-adjust.rst

```diff
@@ -2,8 +2,8 @@ pref-adjust
 ===========
 
 .. dfhack-tool::
-    :summary: Set the preferences of a dwarf to an ideal.
-    :tags: fort armok units
+    :summary: Get the preferences of a dwarf, Set the preferences of a dwarf to a designated profile.
+    :tags: fort armok units preferences
```

Comment:
```suggestion
    :tags: fort armok units
```

Reply:
You can only choose from the existing set of tags (documented here: https://docs.dfhack.org/en/latest/docs/Tags.html )